lib/src/presentation/view/movie_list.dart

```diff
@@ -1,121 +1,78 @@
 import 'package:flutter/material.dart';
 
-import '../custom_widgets.dart';
+import '../../core/state/state.dart';
 import '../../core/util/ui_constants.dart';
-import '../../data/repository/movie_repository.dart';
 import '../../domain/entity/movie.dart';
+import '../bloc/i_bloc_genres.dart';
+import '../bloc/i_bloc_top_rated.dart';
+import '../widget/back_button.dart';
+import '../widget/custom_app_bar.dart';
+import '../widget/custom_drawer.dart';
+import '../widget/list_top_rated.dart';
 
 class MovieList extends StatefulWidget {
-  const MovieList({super.key});
+  const MovieList({super.key, required this.blocTop, required this.blocGenres});
```

Comment:
trailing comma

Reply:
Review the naming, this could be called topRatedBloc and genresBloc

Reply:
done